src/components/select/select.ts

```diff
@@ -49,10 +49,12 @@ export interface SelectChange {
   role: ariaRoleOnHost ? 'listbox' : null,
 })
 export class SbbSelectElement extends SbbUpdateSchedulerMixin(
-  SbbDisabledMixin(SbbNegativeMixin(LitElement)),
+  SbbDisabledMixin(SbbNegativeMixin(SbbOverlayBaseElement)),
 ) {
   public static override styles: CSSResultGroup = style;
-  public static readonly events = {
+
+  // FIXME using { ...super.events, backClick: 'requestBackAction' } breaks the eslint missing-component-documentation-rule
```

Comment:
Same comment again, we should try adapting our rules :)

Reply:
Added a link to https://github.com/sbb-design-systems/lyne-components/issues/2600 in the fixme description